CONTRIBUTING.md

```diff
@@ -126,6 +126,12 @@ Changes:
 * etc...
 ```
 
+##  Publish `israeli-bank-scrapers-core` to NPM.
+1. Make sure everything is committed. 
+2. Run `npm run prepare-core`. The script will change the name inside `package.json` to `prepare-israeli-bank-scrapers-core`, change puppeteer dependency to `puppeteer-core`, reinstall dependencies and rebuild the library. 
+3. Run `npm publish`. This will publish `prepare-israeli-bank-scrapers-core` package.
+4. Run `npm run reset` to reset the changes.
+
```

Comment:
> I'm still confused about the steps needed to publish each variation - would be better to have just one script for publishing each variation (including reset at the end after publishing core) 

@eshaham to answer both this comment and the general one quoted above:
As we are "mimicking" a behavior of `puppeteer` library I used that library as a reference.  They chose to provide manual guide for deploying instead of automating it. See  [puppeteer CONTRIBUTIN.md](
https://github.com/GoogleChrome/puppeteer/blob/master/CONTRIBUTING.md#releasing-to-npm) maybe they have a CI process that run that for them. I don't mind creating a script to automate it if you want.

Reply:
@esakal yes, we should probably have a script for publishing the core variant